server/src/main/java/com/example/tyfserver/donation/domain/Donation.java

```diff
@@ -15,6 +15,8 @@
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class Donation extends BaseTimeEntity {
 
+    public final static long exchangeableDayLimit = 7;
```

Comment:
Donation.exchangeableDayLimit 는 상수인 것 같은데,   `Donation.EXCHANGEABLE_DAY_LIMIT`으로 바꾸는게 좋지 않을까? 

그거 말고도 static이랑 final 위치가 바뀐 것 같아 ㅋㅋㅋ ! 
```suggestion
    public static final long EXCHANGEABLE_DAY_LIMIT = 7;
```